data/genesis/zmlocalconfig/basic.rb

```diff
@@ -100,7 +101,7 @@ def lcXml(data, tagName)
     mcaller.pass = data[0] == 0 && data[1] =~ /#{File.join(Command::ZIMBRAPATH, 'data', 'tmp')}/
   end,
 
-  v(ZMProv.new('ms', '`zmhostname`', 'zimbraserviceEnabled', 'convertd')) do |mcaller, data|
+  v(ZMProv.new('ms', adminAccount, 'zimbraserviceEnabled', 'convertd')) do |mcaller, data|
```

Comment:
Is this command right? zmprov ms will expect server as argument.

Reply:
Typo :( my bad. Fixed it now.